xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -786,7 +786,7 @@ void CActiveAE::StateMachine(int signal, Protocol *port, Message *msg)
             {
               AEDelayStatus status;
               m_stats.GetDelay(status);
-              m_extDrainTimer.Set(status.GetDelay() * 1000);
+              m_extDrainTimer.Set(status.GetDelay() * 1000 + 500);
```

Comment:
No - please! This drives the state machine and it's signals nuts.

Reply:
The 500 fell from heaven. From what I see the only reason it has, is to delay Configure() call by 500 ms when transitioned in IDLE state. The real question is: why does idle configure itself with no one want to play audio.

Reply:
ok @peak3d drop that change. @fritsch bothered me for quite some time, that I'll fix it properly :)